l2/system-contracts/TransactionHelper.sol

```diff
@@ -166,6 +196,9 @@ library TransactionHelper {
 			);
 	}
 
+
+	/// @notice Encode hash of the EIP1559 transaction type.
+	/// @return keccak256 of the serialized RLP encoded representation of transaction
     function _encodeHashEIP1559Tx(Transaction calldata _transaction) private view returns (bytes32) {
```

Comment:
can also avoid `tx` abbreviation here

Reply:
Sure, we already implemented that. But that contracts version in on the audit now. We will deployed them after the audit will be finished